File: Modul 1/Soal 8/PRAK108-NIM-Nama.c
#include <stdio.h>
#include <math.h>

int main(){
    int putaran = 5;
    double jarak = 14.0;
    double keliling = jarak / putaran;
    double radius = keliling / (2 * M_PI);

    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %d Putara\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n", jarak);
    printf("\n");
    printf("Jawaban :\n");
    printf("jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", radius);
}

File: Modul 1/Soal 08/PRAK108-NIM-Nama.py
import math

jarak = 14
putaran = 5
keliling = jarak / putaran
radius = keliling / (2 * math.pi)

print("Diketahui :")
print("Pak Dengklek mengelilingi taman =", putaran)
print("Jarak tempuh Pak Dengklek =", jarak)
print()
print("Jawaban :")
print("Jari-jari taman yang dikelilingi Pak Dengklek adalah", f"{radius:.2}")